docs/core/preview3/tools/dotnet-build.md

```diff
@@ -23,33 +23,37 @@ dotnet-build -- Builds a project and all of its dependencies
 
 ## Synopsis
 
-`dotnet build [--help] [--output]  [--framework]  
-    [--configuration]  [--runtime] [--version-suffix]
-    [--build-profile]  [--no-incremental] [--no-dependencies]
-    [<project>]`
+```
+dotnet build <project>
+dotnet build [-h|--help]
+dotnet build [-o|--output] [-f|--framework] [-c|--configuration] [-r|--runtime]
+dotnet build [--version-suffix] [--no-incremental] [--no-dependencies]
+```
+
+
 
 ## Description
+The `dotnet build` command builds the project and its dependencies into a set of binaries. The binaries are the symbol files used for debugging (having a `*.pdb` extension) as well as the project's code in Intermediate Language (IL) with an `*.dll` extension. Additionally, a JSON file that lists out the dependencies of the application with the `*.deps` extension will be produced. Finally, a `runtime.config.json` file will be produced as well. This file specifies which shared runtime and version the built code will run against. 
+
+If the project has third-party dependencies, such as libraries from NuGet, these will be resolved from the NuGet cache and will not be available with the project's built output. With that in mind, the product of `dotnet build` is not ready to be transferred to another machine to run. This is in contrast to the behavior of .NET Framework in which building an executable project (an application) will produce an output that is possible to run on any machine that has .NET Framework installed. In order to get a similar experience in .NET Core, you have to use the [dotnet publish](dotnet-publish.md) command. More information about this can be found in the [.NET Core Application Deployment](../deploying/index.md) document. 
 
-The `dotnet build` command builds multiple source file from a source project and its dependencies into a binary. 
-By default, the resulting binary is in Intermediate Language (IL) and has a DLL extension. 
-`dotnet build` also drops a `*.deps` file which outlines what the host needs to run the application.  
+Building requires the existence of an asset file (a file that lists all of the dependencies of your application), which means that you have to run [`dotnet restore`](dotnet-restore.md) prior to building the project. Lack of the assets file manifests as the inability of the tooling to resolve reference assemblies. 
 
-Building requires the existence of an asset file (a file that lists all of the dependencies of your application), which 
-means that you have to run [`dotnet restore`](dotnet-restore.md) prior to building your code.
+`dotnet build` uses MSBuild to build the project, thus it support both parallel builds and incremental builds. Please refer to [MSBuild documentation]() to get more information on those topics. 
```

Comment:
it support -> it supports

missing link

Please refer to [MSBuild documentation]() to get more information on those topics ->
For more information, see ....